distribution.md

```diff
@@ -228,3 +230,89 @@ The Run Image SHOULD contain the following Labels on the image configuration:
 Where,
 
 `<Target ID>` is an identifier specified on the runtime image that MAY be used to apply target-specific logic.
+
+### Builder
+
+The following specifies the artifact format for a buildpacks builder.
+
+A builder is an OCI Image that provides a distributable build environment.
+
+A platform supporting builders SHOULD allow users to configure the build environment for a provided builder.
+
+#### General Requirements
+
+The builder image MUST contain an implementation of the [lifecycle](#lifecycle), and a [build-time](#build-image) environment, and MAY contain [buildpacks](#buildpackage). Platforms SHOULD use builders to ease the build process.
+
+#### Filesystem
+
+A builder MUST have the following directories/files:
+
+* `/cnb/order.toml` &rarr; As defined in the [platform specification](https://github.com/buildpacks/spec/blob/main/platform.md#ordertoml-toml)
```

Comment:
I suggest that we remove this requirement entirely given that we haven't thought about how to handle the case where we change or remove this file between platform APIs. The platform can always pass in a valid `order.toml` for it's preferred platform API, created from the label if it wants to use the builder-provided default.

Reply:
I was in the [same boat](https://github.com/buildpacks/spec/pull/271/files#r760387860) for this and other reasons. I eventually caved in to minimize the turbulence. Happy to revisit but would like to take @jabrown85's [points](https://github.com/buildpacks/spec/pull/271/files#r767823046) into consideration.

Reply:
I'd love to defer this to a separate talking point, if you feel strongly we can even block the release of the distribution spec on it.

Reply:
Punted to https://github.com/buildpacks/spec/issues/291